modules/light-clients/08-wasm/types/client_state.go

```diff
@@ -220,9 +220,16 @@ func call[T ContractResult](ctx sdk.Context, clientStore sdk.KVStore, cs *Client
 	if err != nil {
 		return result, errorsmod.Wrapf(err, "call to wasm contract failed")
 	}
+	// Only allow Data to flow back to us. SubMessages, Events and Attributes are not allowed.
```

Comment:
though we can not be as granular (and just return a single err message), doing so allows us to easily allow one of these by simply removing relevant code.